fakesqs/metadata.rb

```diff
@@ -0,0 +1,6 @@
+name 'fakesqs'
+
+supports 'ubuntu'
+
+depends 'ruby-brightbox'
```

Comment:
Good point.  I built the recipe while doing a full on kitchen/chef/knife tutorial and this is likely **not** a requirement in our full cookbook chain.  I will test and revamp this morning.